tests/manage/pv_services/test_rwo_pvc_fencing_unfencing.py

```diff
@@ -791,26 +765,27 @@ def test_rwo_pvc_fencing_node_prolonged_network_failure(
                 f"App pod with name {pod_obj.name} did not reach Running state"
             )
 
-        # Wait for mon and osd pods to reach Running state
-        selectors_to_check = [constants.MON_APP_LABEL, constants.OSD_APP_LABEL]
-        for selector in selectors_to_check:
-            assert ceph_cluster.POD.wait_for_resource(
-                condition=constants.STATUS_RUNNING, selector=selector,
-                resource_count=3, timeout=1800, sleep=60
-            ), (
-                f"3 expected pods with selector {selector} are not in Running state"
-            )
+        if not external_mode:
+            # Wait for mon and osd pods to reach Running state
+            selectors_to_check = [constants.MON_APP_LABEL, constants.OSD_APP_LABEL]
+            for selector in selectors_to_check:
+                assert ceph_cluster.POD.wait_for_resource(
+                    condition=constants.STATUS_RUNNING, selector=selector,
+                    resource_count=3, timeout=1800, sleep=60
+                ), (
+                    f"3 expected pods with selector {selector} are not in Running state"
+                )
 
-        if ceph_cluster.mon_count == 3:
-            # Check ceph health
-            toolbox_status = ceph_cluster.POD.get_resource_status(
-                ceph_cluster.toolbox.name
-            )
-            if toolbox_status == constants.STATUS_TERMINATING:
-                ceph_cluster.toolbox.delete(force=True)
+            if ceph_cluster.mon_count == 3:
```

Comment:
same

Reply:
This was intentional. We don't want to check Ceph Health in case of more than 3 mons, since some mons might not be in Running state when we have fewer OCS labelled nodes